src/ast/transform/MaterializeAggregationQueries.cpp

```diff
@@ -197,11 +242,10 @@ bool MaterializeAggregationQueriesTransformer::materializeAggregationQueries(
         });
     });
     return changed;
-}
+}  // namespace souffle
```

Comment:
```suggestion
}
```
I think this was added in by mistake.